.github/workflows/trivy-scheduled-scan.yaml

```diff
@@ -0,0 +1,76 @@
+name: Trivy Scheduled Scans
+
+on:
+  # Scan schedule is the same as codeql-analysis job.
+  schedule:
+    - cron: '10 18 * * 2'
+
+jobs:
+  scan_MAIN:
+    if: ${{ github.repository == 'CrunchyData/postgres-operator' }}
```

Comment:
🔧  want this on the `scan_REL_4_7` job too?

Reply:
Ouch. Good catch!